kofta/src/app/utils/useTypeSafeTranslation.ts

```diff
@@ -1,15 +1,16 @@
 import { useTranslation } from "react-i18next";
 import { TranslationKeys } from "../../generated/translationKeys";
 
-interface DateTranslationType {
+interface VariableTranslationType {
```

Comment:
Why is this interface name changed?

Reply:
Before you could only pass in dates as variables to i18next. Since I added a "count", with the type of number, the name didn't feel applicable anymore.

Reply:
Understood.